client/src/app/pages/vulnerability-list/vulnerability-context.tsx

```diff
@@ -88,15 +88,15 @@ export const VulnerabilitySearchProvider: React.FunctionComponent<
     getHubRequestParams({
       ...tableControlState,
       hubSortFieldKeys: {
-        identifier: "identifier",
+        name: "name",
         severity: "average_score",
       },
     })
   );
 
   const tableControls = useTableControlProps({
     ...tableControlState,
-    idProperty: "identifier",
+    idProperty: "name",
```

Comment:
This should remain "identifier".

- The "idProperty" points to the name of the field of an object
 E.g. Let's say the hook `useTableControlProps` is receiving a list of objects like:

```json
[
  {
     "id": 1,
     "name": "Maria",
     "surname": "Something"
  }
]
```

Then `idProperty` should point to the name of an unique field of the object being passed to the hook. Actually the IDE should be able to help there suggesting which are the available options.

In this case since we are loading Vulnerabilities then we need to pass the name of the unique (primary key) of the object, which in the case of Vulnerabilities is `identifier`

![image](https://github.com/user-attachments/assets/156ee759-cc28-45d5-acff-9088d9cd57a6)

Reply:
Thanks @carlosthe19916, I assumed we were also changing the structure, or keys, to go from `name` to `identifier` but it seems it's just for the rendering/display, which makes sense as it seems those types are automatically generated anyway.